packages/fields-wysiwyg-tinymce/README.md

```diff
@@ -0,0 +1,32 @@
+---
+section: field-types
+title: Wysiwyg
+---
+
+# WYSIWYG HTML Field powered by TinyMCE Editor
+
+This field inserts a string path into your schema based on the `Text` field type implementation, and renders a WYSIWYG editor in the Admin UI using [TinyMCE](https://www.tiny.cloud/)
```

Comment:
```suggestion
This field inserts a string path into your schema, and renders a WYSIWYG editor in the Admin UI using [TinyMCE](https://www.tiny.cloud/)
```

The internal implementation probably isn't really important to most users.

Reply:
I actually think this is useful information - and I'd like to link to the `Text` documentation because that will eventually describe the graphql schema / filters / etc that are relevant for this field as well.

Reply:
> that will eventually describe the graphql schema / filters / etc that are relevant for this field as well.

Great point 👍